### PR TITLE
fix favicon

### PR DESCRIPTION
**Pull request by geeknees**
## 🛠️ Fixes Issue

- Fixed issue with favicons not displaying

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

Related but different corrective actions
https://github.com/FrancescoXX/free-Web3-resources/issues/123

### CODE REVIEW

**Comment by Atharv181** [comment]
I think it is not required to change in code because it is working fine with previous code.
Problems arrived when we host the site using GitHub pages.

**Comment by geeknees** [comment]
Yes, the migration to GitHub pages seems to be a problem.

The URL for GitHub pages is as follows, with the repository name under the top directory and the content under that.
https://francescoxx.github.io/free-Web3-resources/

As it stands, the references are

`https://francescoxx.github.io/images/favicon_io/favicon.ico`

but it should be

`https://francescoxx.github.io/free-Web3-resources//images/favicon_io/favicon.ico`

to see the reference.

File: index.html [hunk 1]
```diff
@@ -3,7 +3,11 @@
   <head>
     <meta charset="UTF-8" />
     <title>FREE Web3 Resources</title>
-    <link rel="icon" type="image/x-icon" href="/images/favicon_io/favicon.ico">
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="/free-Web3-resources/images/favicon_io/favicon.ico"
+    />
```
Comment (vikasganiga05) [inline index.html:10]: The following code will fix it :point_down: 
```suggestion
    <link
      rel="icon"
      type="image/x-icon"
      href="./images/favicon_io/favicon.ico"
    />
```

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Hi, no need to change the path. You need to prefix the path with a `.`(dot). Like this :point_down: 

`./images/favicon_io/favicon.ico`

File: index.html (hunk 1 above)
Comment (geeknees) [reply index.html]: Yes, we can solve this problem with relative paths. Thank you.

**Review by FrancescoXX — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (vikasganiga05) [reply index.html]: It's my pleasure to help :slightly_smiling_face:

**Pull request merged**